Formula/gopls.rb

```diff
@@ -5,6 +5,11 @@ class Gopls < Formula
   sha256 "08d477a7c35021ec5d8950e25e1fcac86d7ec0ce8a421c20d932029e00efb1d8"
   license "BSD-3-Clause"
 
+  livecheck do
+    url "https://github.com/golang/tools/releases/latest"
```

Comment:
```suggestion
    url :stable
    strategy :github_latest
```

Reply:
You are right, my brew is not up to date, that is why my livecheck got messed up.